webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -50,7 +50,12 @@ class BreadcrumbBar extends React.Component {
 
     return (
       <div className="breadcrumb-bar">
-        <Breadcrumb title items={breadcrumbItems} isTitle={isTitle}>
+        <Breadcrumb
+          title
+          items={breadcrumbItems}
+          isTitle={isTitle}
+          titleAddition={this.props.titleAddition}
```

Comment:
1. Can you put it with all the other props in the `const { ... } = this.props`?

2. Can you add `propType` and `defaultProp`?